src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
+        System.out.println(ball1 + "\n" + ball2 + "\n" + ball3);
```

Comment:
numbers in naming are often a sign you're doing something wrong, here you should use a for loop instead. Make sure for loop body is just one line